index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/8f283d01ad.js" crossorigin="anonymous"></script>
+    <title>Daron Wiafe (Software Developer)</title>
+</head>
+<body>
+    <div id="wrapper">
+        <nav class="portfolio_nav">
+            <img src="daron.jpeg" alt="author of the website">
+            <a href="#" class="menu_dropdown"><i class="fas fa-bars fa-2x"></i></a>
+            <ul class="links">
+                <li><a>Portfolio</a></li>
+                <li><a>About</a></li>
+                <li><a>Contact</a></li>
+            </ul>
+        </nav>
+        <header class="intro">
+            <h1 class="developer">Daron Wiafe</h1>
+            <h3 class="occupation">Software Developer</h3>
+        </header>
```

Comment:
- Kindly, the header by default is used at the top and includes the NAV tag, otherwise use the SECTION tag.
- For accessibility, it is recommended to use the order of importance and the titles, so H1 is followed by H2.